enterprise/payara.tooling/src/org/netbeans/modules/payara/tooling/server/parser/JvmConfigReader.java

```diff
@@ -201,7 +201,7 @@ public boolean isMonitoringEnabled() {
     public static class JvmOption {
 
         public final String option;
-        public final Optional<String> vendor;
+        public final Optional<String> vendorOrVM;
```

Comment:
This is also failing sigtests -

```
Class org.netbeans.modules.payara.tooling.server.parser.JvmConfigReader$JvmOption

  "E1.2 - API type removed" : field public final java.util.Optional org.netbeans.modules.payara.tooling.server.parser.JvmConfigReader$JvmOption.vendor

  warn: "W5.6 - Adding fields" : field public final java.util.Optional org.netbeans.modules.payara.tooling.server.parser.JvmConfigReader$JvmOption.vendorOrVM
```

Reply:
Keep the original field name. You may change its runtime meaning (that of course isn't going to be detected by signature tests, but only by failures in the runtime), or you may introduce new field and possibly deprecate the old one.

Lesson to learn: don't expose classes that aren't needed in your API. Don't expose fields, but rather getters. Chapter 5: [Don't Expose More Than You Want](http://wiki.apidesign.org/wiki/Do_Not_Expose_More_Than_You_Want).